discovery-provider/src/api_helpers.py

```diff
@@ -61,7 +52,11 @@ def response_dict_with_metadata(response_entity=None):
     response_dictionary['latest_indexed_block'] = (int(latest_indexed_block) if latest_indexed_block else None)
     response_dictionary['latest_chain_block'] = (int(latest_chain_block) if latest_chain_block else None)
     response_dictionary['version'] = disc_prov_version
-    response_dictionary['owner_wallet'] = shared_config['delegate']['owner_wallet']
+    response_dictionary['delegateWallet'] = shared_config['delegate']['owner_wallet']
+
+    signature, timestamp = generate_signature_and_timestamp(response_dictionary)
+    response_dictionary['signature'] = signature
+    response_dictionary['timestamp'] = timestamp
```

Comment:
I think we're still duplicating the logic for this per my prior comment - the signature process has to know the key is string "timestamp", then we have to know / insert the same key here too or risk breaking the whole signature (if they are one character off for example)

the object editing should be consolidated to one place imo - could we insert the timestamp here before the function call, then call generate_signature to get the signature (since it doesn't do the timestamp thing), then insert that in?